server/py/services/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -110,18 +111,21 @@ async def create_model_endpoint(
             )
             try:
                 logger.info("Getting model object from db")
+                db_artifact = framework.utils.singletons.db.get_db().read_artifact(
+                    db_session,
+                    key,
+                    tag,
+                    iteration,
+                    project,
+                    tree,
+                    uid,
+                    as_record=True,
```

Comment:
please add comment - need as record to add the model id foreign key
though I just commented I think it should be removed